src/main/java/persistence/model/EntityTable.java

```diff
@@ -36,7 +36,22 @@ public List<EntityColumn> getPrimaryColumns() {
         return tableColumns.getPrimaryColumns();
     }
 
+    public List<EntityColumn> getNonPrimaryColumns() {
+        return tableColumns.getNonPrimaryColumns();
+    }
+
     public void setColumns(List<EntityColumn> columns) {
         tableColumns.setColumns(columns);
     }
+
+    public List<EntityColumn> getInsertableColumns(EntityTable table) {
+        return table.isPrimaryColumnsValueSet()
+                ? table.getColumns()
+                : table.getNonPrimaryColumns();
+    }
```

Comment:
`getInsertableColumns`의 메서드 네이밍은 INSERT 문을 생성하기 위해서만 사용 되어야할 것처럼 느껴지는데요!
조금 더 추상화된 메서드 네이밍을 통해 간접적인 의존성을 낮춰보는 것도 좋을 것 같습니다 :)

```suggestion
    public List<EntityColumn> getNormalColumns(EntityTable table) {
        return table.isPrimaryColumnsValueSet()
                ? table.getColumns()
                : table.getNonPrimaryColumns();
    }
```

Reply:
`getActiveColumns` 로 수정했습니다~